src/dft/src/dft.tcl

```diff
@@ -47,7 +47,7 @@ proc insert_dft {} {
 
 proc set_dft_config { args } {
   sta::parse_key_args "set_dft_config" args \
-    keys {-max_length} \
+    keys {-max_length -clock_mixing} \
```

Comment:
The README should have command documentation.  Having that would help me to know what this option is for.

Reply:
I updated the README with the current status of the tool, thanks!